lib/main.ml

```diff
@@ -23,6 +23,8 @@
 
 open Lwt
 
+[@@@warning "-3"]
```

Comment:
why is this needed?

Reply:
@pascutto can correct me if I'm wrong, but it was because the use of `Lwt_sequence` is deprecated. Thanks to #42 we can remove that while rebasing.